colors/onedark.vim

```diff
@@ -577,6 +577,9 @@ call s:h("GitGutterDelete", { "fg": s:red })
 call s:h("ALEError", { "fg": s:red, "gui": "underline", "cterm": "underline" })
 call s:h("ALEWarning", { "fg": s:yellow, "gui": "underline", "cterm": "underline" })
 call s:h("ALEInfo", { "gui": "underline", "cterm": "underline" })
+call s:h("ALEErrorSign", { "fg": s:red })
+call s:h("ALEWarningSign", { "fg": s:yellow })
+call s:h("ALEInfoSign", { })
```

Comment:
Thanks for opening this PR!

I'm curious whether this line with no colors specified has any actual effect, or if you just included it for completeness?

Reply:
Yes, I added it for completeness.

Variables for line and sign differ in underlining, all other properties are the same. In this case, `fg`.

With this line info sign has the following highlighting:
```
:hi ALEInfoSign
ALEInfoSign    xxx cleared
```

It looks valid.